public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="h-full bg-white">
```

Comment:
Why do we have this changes?

Reply:
it requires in documentation as you can see -
https://tailwindui.com/components/application-ui/forms/sign-in-forms


> Why do we have this changes?